Travis/js/main.js

```diff
@@ -0,0 +1,219 @@
+
+
+
+
+// A light switch that can be either on or off. Booleans
+
+let light = false;
+
+// A user's email address. Objects  
+ 
+let email = { 
+	name: 'unknown',
+	email: 'email@unknown.com'
+}
+
+// A spaceship with a hull, laser blasters, tractor beam, and warp drive. String
+
+let spaceship = ('hull', 'laser blasters', 'tractor beam', 'warp drive')
```

Comment:
-0.75 for this
0.25: array or object are better answers (although, string is sort of ok)
0.5: this is not valid syntax.  `spaceship` will just be "warp drive"